packages/@sanity/types/src/schema/definition/schemaDefinition.ts

```diff
@@ -110,6 +113,14 @@ export type TypeAliasDefinition<
   validation?: SchemaValidationValue
   initialValue?: InitialValueProperty<any, any>
   preview?: PreviewConfig
+
+  components?: {
```

Comment:
Reinstates components with any props for TypeAliasDefinition, ie, the fallback case when we cannot determine an exact type.